debian/changelog

```diff
@@ -1,3 +1,11 @@
+cloud-init (22.4.1-0ubuntu0~22.10.1) kinetic; urgency=medium
```

Comment:
We'll need to sort the actual package version. In this case, we released `22.4-0ubuntu4` to lunar, so we can't release a version `22.4.1-0ubuntu0~22.10.1` to kinetic because  pkg versioning won't be an upgrade path
```bash
dpkg --compare-versions 22.4-0ubuntu4 gt 22.4.1-0ubuntu0~22.10.1 || echo "NOT A VALID UPGRADE"
NOT A VALID UPGRADE
```

Reply:
Release of 22.4.1-0ubuntu1 has been published to lunar. So we are good on versioning now.